### PR TITLE
Added Login Page With Side Image Template

### LINKED ISSUE #50: Make a login page with side image template
Hello @Diptenusarkar , I want to add a login page with side image template.
This template will have validations upon login and also a "sign in with google" button.
If you find this appropriate then please assign it to me.

### PR DESCRIPTION
**Pull request by Sumedh-byte**
Hello @Diptenusarkar , I have added the login page with side image template. 
The login form will have all validations plus a forgot password and remember me option.
The template will also notify what is wrong in case of username and password.
This will close #50 
Thank you.

### CODE REVIEW

**Comment by Diptenusarkar** [comment]
Wait I need Check.

**Comment by Sumedh-byte** [comment]
Is it okay?
Is my PR merged? because on Hacktober site it is showing status as accepted and the countdown is started.

**Review by Diptenusarkar — APPROVED** [review]
Good Job

**Comment by Sumedh-byte** [comment]
Hello @Diptenusarkar , as GitHub code scanner is giving some warnings, I am resolving the issue one more time. Please consider my request and later please merge the pull request.

File: Login Page/validation.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+function check(event){
+    event.preventDefault();
+    var email1=document.getElementById('mail');
+    var passd1=document.getElementById('pswd');
+    var mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```
[the remark this answers is not in the document]
Comment (Sumedh-byte) [reply Login Page/validation.js]: Resolved this issue

[the remark this answers is not in the document]
Comment (Sumedh-byte) [reply Login Page/validation.js]: Resolved this issue

**Review by Sumedh-byte — COMMENTED** [review]
All issues solved

**Review by Diptenusarkar — APPROVED** [review]

**Pull request merged**